contracts/package.json

```diff
@@ -18,7 +18,7 @@
     "prepublishOnly": "pnpm compile && ./scripts/prepublish_generate_abi_folder",
     "publish-beta": "pnpm publish --tag beta",
     "publish-prod": "pnpm publish --tag latest",
-    "solhint": "solhint --max-warnings 2 \"./src/v0.8/**/*.sol\""
+    "solhint": "solhint --max-warnings 3 \"./src/v0.8/**/*.sol\""
```

Comment:
Fix the issue, don't increment the counter. If the behaviour is desired add a solhint ignore line

Reply:
I just reverted it back. This change was not needed as the solhint issues was on our I*RegistryMaster.sol, which should ignore solhint check (https://github.com/smartcontractkit/chainlink/pull/13504/files#diff-bf3ee5318018ff95ee8303e7d1ed0c0855a9520d003ba32928266ec74e915e0d). So this line of change is not needed at all. I just changed it back